tests/request_parsing_tests.rs

```diff
@@ -41,22 +47,32 @@ fn test_create_window1() {
         minor_opcode: 0x18,
         remaining_length: 10,
     };
-    let body = &[
-        0x1, 0x0, 0xc0, 0x5, 0x13, 0x5, 0x0, 0x0, 0x7b, 0x4, 0x34, 0x1, 0xf5, 0x3, 0x3b, 0x3, 0x0,
-        0x0, 0x1, 0x0, 0x47, 0x0, 0x0, 0x0, 0x1a, 0x0, 0x0, 0x0, 0xf0, 0xf1, 0xf2, 0xff, 0x0, 0x0,
-        0x0, 0x0, 0x1, 0x0, 0x0, 0x0,
-    ];
-    let r = CreateWindowRequest::try_parse_request(header, body).unwrap();
+    let mut body = vec![];
+    add_ne!(body, 0x05c00001u32);
+    add_ne!(body, 0x00000513u32);
+    add_ne!(body, 0x047bu16);
+    add_ne!(body, 0x0134u16);
+    add_ne!(body, 0x03f5u16);
+    add_ne!(body, 0x033bu16);
+    add_ne!(body, 0x0000u16);
+    add_ne!(body, 0x0001u16);
+    add_ne!(body, 0x00000047u32);
+    add_ne!(body, 0x0000001au32);
+    add_ne!(body, 0xfff2f1f0u32);
+    add_ne!(body, 0x00000000u32);
+    body.push(0x01);
+    body.extend(&[0x00, 0x00, 0x00]); // Final padding.
+    let r = CreateWindowRequest::try_parse_request(header, &body).unwrap();
```

Comment:
Does anyone have access to a big endian machine or a good idea on how we could test this on CI? The best idea that I have so far would be `qemu` and I bet that would be quite slow.

Alternatively, I am fine with just assuming this works. That approach also worked so far. :-)